aea/cli/utils/click_utils.py

```diff
@@ -252,19 +254,21 @@ def wrapper(f: Callable) -> Callable:
     return wrapper
 
 
-def remote_registry_flag(mark_default: bool = True) -> Callable:
+def remote_registry_flag(
+    mark_default: bool = True,
+    default_registry: str = REMOTE_IPFS,
+) -> Callable:
     """Choice of one flag between: '--ipfs/--http'."""
 
-    default_registry: Optional[str] = (
+    default_registry = default_registry or (
```

Comment:
Same here, how will the `or` ever evaluate if `default_registry` is of type `str`?